content/applications/inventory_and_mrp/manufacturing/workflows/byproducts.rst

```diff
@@ -0,0 +1,75 @@
+===========
+By-Products
+===========
+
+.. |MO| replace:: :abbr:`MO (manufacturing order)`
+.. |BoM| replace:: :abbr:`BoM (bill of materials)`
+.. |BoMs| replace:: :abbr:`BoMs (bills of materials)`
+
+When manufacturing certain products, it is common to be left with residual materials, in addition to
+the finished product. These materials are know as *by-products*. By specifying the by-products
+created during manufacturing on a product's bill of materials (BoM), the quantity of by-products
+on-hand is tracked by Odoo.
+
+.. example::
+   Manufacturing a *rocking chair* requires ten pieces of wood. During production, five pieces of
+   *scrap wood* are created in addition to the Rocking Chair. By designating the scrap wood as a
+   by-product on the rocking chair's BoM, Odoo tracks the on-hand count of scrap wood, as well as
+   rocking chairs.
+
+Configuration
+=============
+
+To specify by-products on a product's |BoM|, the *By-Products* setting must be enabled. To do so,
+navigate to :menuselection:`Manufacturing app --> Configuration --> Settings`, and enable the
+:guilabel:`By-Products` checkbox, located under the :guilabel:`Operations` heading.
```

Comment:
Feel free to disregard if this isn't common practice, but I had to save the changes after enabling the checkbox before moving on to the next section. Would you mention this here?